examples/analyse_meshgrid_voidanalysis_MFI.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+'''
+
+An example showing how to plot the total volume unoccupied by molecular vdW spheres.
+
+A WORD OF WARNING - this script shows the total volume that can be occupied by a probe sphere
+of a set radius, but does not differentiate between volumes that are accessible/inaccessible
+to the probe. More sophisticated techniques such as Delauney triangulation required.
+
+'''
+
+def test_void_find():
+
+    from carmm.analyse.meshgrid.meshgrid_meshobject import Mesh
+    from carmm.analyse.meshgrid.meshgrid_functions import atom_mesh_build_mask, distance_meshgrid2point
+    from carmm.analyse.meshgrid.meshgrid_void import void_find, void_build_mask, void_analysis
+    import matplotlib.pyplot as plt
+    import numpy as np
+    from ase.io import read
+
+    # Initialise the Mesh object, with a default underlying meshgrid of 50x50x50 grid points.
+    mfi_mesh = Mesh(np.array([20.22614449, 19.82125040, 13.36948553, 90, 90, 90]),
+                    nx = 20, ny = 20, nz = 20, pbc=[1, 1, 1])
```

Comment:
What if there is a mismatch between Atoms.pbc and Mesh.pbc?

Reply:
True - have included a function that checks whether PBCs match (and an option to swtich it off for future bug fixing) wherever atoms objects are used.